src/startup_scripts.rs

```diff
@@ -1037,7 +1030,6 @@ bench-tps)
   clientCommand="\
     solana-bench-tps \
       --sustained \
-      --threads $threadCount \
```

Comment:
Did you mean to remove this?

Reply:
ya i did. because the way it was setup, you couldn't actually set the threads with cli flags. it would just take the number of cores and if number of cores was greater than 4, it would set to 4. And the default thread count in bench-tps is 4 anyway. So I took it out. now you can pass in `threads=4` into `--bench-tps-args`

Reply:
gotcha, thanks!